rir/src/compiler/pir/instruction.h

```diff
@@ -585,12 +586,18 @@ class FLIE(StVar, 2, Effect::None, EnvAccess::Write) {
     void printArgs(std::ostream& out, bool tty) override;
 };
 
-class FLI(Branch, 1, Effect::None, EnvAccess::None) {
+// Common interface to all branch instructions
+class BranchInstruction {
```

Comment:
can we call this `BranchingInstruction` to avoid confusion with the actual `Branch` Instructions